src/domain/app/footer/__tests__/__snapshots__/Footer.test.tsx.snap

```diff
@@ -76,7 +76,7 @@ exports[`Footer matches snapshot 1`] = `
             © 
             Copyright
              
-            2024
+            2025
```

Comment:
The test would be good to fix in a long term way so that this test wouldn't break come 2026. Not necessarily in this PR though.